src/Fantomas/FormatConfig.fs

```diff
@@ -39,6 +40,7 @@ type FormatConfig =
           ReorderOpenDeclaration = false
           SpaceAroundDelimiter = true
           KeepNewlineAfter = false
+          BracketOnSeparateLine = false // TODO: add to cli
```

Comment:
TODO?

Reply:
That has been done, the comment needs to removed.